matRad_calcLateralParticleCutOff.m

```diff
@@ -79,14 +79,19 @@
          
         if strcmp(machine.meta.dataType,'singleGauss')
                     
-                    Sigma1 = machine.data(energyIx).sigma(Idx(j));
+                    Sigma1 = sqrt(machine.data(energyIx).sigma(Idx(j))^2 + machine.data(energyIx).initFocus.sigma(1)^2);
                     y_r = SG(r_mid,Sigma1);
                  
         elseif strcmp(machine.meta.dataType,'doubleGauss')
-            
+                    
+                    % use biggest focus in stf struct ?
+                    focusIdx = 2;
```

Comment:
The focus index should be a function of the energy.
We could recompute the selected focus size for the considered energy. This, however, means that we need to do this twice and also specify the parameter for that twice or pass it on as input for this function.